application/views/satellite/flightpath.php

```diff
@@ -46,12 +45,17 @@
 					echo '>' . $sat->satname . '</option>'."\n";
 				} ?>
 			</select>
+		</form>
 
+		<?php } else { ?>
+			<?= "No satellites with TLE found. Please update via CRON or satellite page. If you have no access to do this, ask your admin!"; ?>
```

Comment:
You forgot the gettext function

```suggestion
			<?= __("No satellites with TLE found. Please update via CRON or satellite page. If you have no access to do this, ask your admin!"); ?>
```